modules/assim.batch/R/pda.emulator.R

```diff
@@ -345,23 +353,24 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
       }
     }
     colnames(m) <- unlist(pname)[prior.ind.all]
-    mcmc.list.tmp[[c]] <- m
+    mcmc.list[[c]] <- m
     
-    jvar.list[[c]] <- mcmc.out[[c]]$jump@history[nrow(mcmc.out[[c]]$jump@history),]
+    #jmp.list[[c]] <- mcmc.out[[c]]$jump
+    resume.list[[c]] <- mcmc.out[[c]]$chain.res
   }
   
   
-  if(length(extension.check)==1 & !run.block){
-    
-    # merge with previous run's mcmc samples
-    mcmc.list <- mapply(rbind, mcmc.list, mcmc.list.tmp, SIMPLIFY=FALSE)
-    settings$assim.batch$iter <- nrow(mcmc.list[[1]])
-    
-  }else{
-    
-    mcmc.list <- mcmc.list.tmp
-
-  }
+  # if(length(extension.check)==1 & !run.block){
```

Comment:
commented out some lines here and there while updating, maybe I better delete these